dashboard/d_graphs.py

```diff
@@ -6,12 +6,15 @@
 
 def create_bubble_chart(df: pd.DataFrame) -> alt.Chart:
     """Returns a bubble chart of sentiment for a source by topic."""
-    return alt.Chart(df).mark_circle().encode(
+
+    color_scale = alt.Scale(domain=['Fox News', 'Democracy Now!'],
```

Comment:
Nice adding contrasting colours to distinguish different source data points better